File: app/src/main/java/com/example/mathalarm/ShowLogs.java
package com.example.mathalarm;

import android.util.Log;
import com.example.mathalarm.Alarms.MathAlarm.MainMathAlarm;

/** @author Konstantyn
 * Created by Администратор on 14.03.2017.
 */

public  class ShowLogs {
    private static final String TAG = "MathAlarm";

    public static boolean LOG_STATUS= MainMathAlarm.LOG_DEBUG_STATUS ? true : false;

    public static void i(String logMessage) {
        Log.i(TAG," "+logMessage);
    }
}

File: app/src/main/java/com/mrkostua/mathalarm/Tools/ShowLogs.kt
package com.mrkostua.mathalarm.Tools

import android.util.Log

/**
 * @author Kostiantyn Prysiazhnyi on 03.12.2017.
 */

object ShowLogs {
    var LOG_STATUS = if (ConstantValues.LOG_DEBUG_STATUS) true else false

    fun log(TAG: String, logMessage: String) {
        if(LOG_STATUS)
        Log.i(TAG, " " + logMessage)
    }

}